exp/internal/controllers/suite_test.go

```diff
@@ -59,3 +60,11 @@ func TestMain(m *testing.M) {
 		SetupReconcilers: setupReconcilers,
 	}))
 }
```

Comment:
@sbueringer  I followed this approach of having fake controller, Please suggest is there any better alternative.
If this approach looks good, Is there any common place I can move this code to avoid duplication across package. Thanks

Reply:
Approach is fine and simple. I don't see a better alternative.

I would say let's add a "fake" sub-package to controllers/external

(we have a similar pattern in some other packages, I think client-go as well)

Reply:
Sure, Thank you. I will update it accrodingly.